BeatSaberHTTPStatus/Plugin.cs

```diff
@@ -119,11 +126,12 @@ public class Plugin : IPlugin {
 				noteCutMapping.Clear();
 
 				statusManager.EmitStatusUpdate(ChangedProperties.AllButNoteCut, "menu");
-			} else if (newScene.name == "GameCore") {
+			}
+			else if (newScene.name == "GameCore") {
```

Comment:
The brackets closing the previous and opening the next block should be on the same line.